get-started/code-experiments/README.md

```diff
@@ -0,0 +1,260 @@
+  `add_noise` is `true`.
+
+- `train.validation_split`: The split ratio for the validation set, reserved
+  from the training set. If this value is `0`, the test set is used for
+  validation. 
+
+- `train.epochs`: Number of epochs to train the network. 
+
+- `train.batch_size`: Batch size for the `model.fit` method. 
+
+- `model.name`: Used to select the model. For `mlp` a simple NN with a single
+  hidden layer is used. For `cnn`, a Convolutional Net with a single `Conv2D`
+  and a single `Dense` layer is used. The parameters for these networks are defined in separate sections below.
+  
+- `model.optimizer`: Can be one of `Adam`, `SGD`, `RMSprop`, `Adadelta`, `Adagrad`, `Adamax`, `Nadam`, `Ftrl`.
+
+- `model.mlp.units`: Number of `Dense` units in MLP.
+
+- `model.mlp.activation`: Activation function for the `Dense` layer. Can be one
+  of `relu`, `selu`, `elu`, `tanh`
+
+- `model.cnn.dense_units`: Number of units in `Dense` layer of the CNN.
+
+- `model.cnn.activation`: The activation function for the convolutional layer.
+  Can be one of `relu`, `selu`, `elu` or `tanh`.
+
+- `model.cnn.conv_kernel_size`: One side of convolutional kernel, e.g., for
+  `3`, a `(3, 3)` convolution applied to the images.
+
+- `model.cnn.conv_units`: Number of convolutional units. 
+
+- `model.cnn.dropout`: Dropout rate between `0` and `1`. 
+
+### Metrics
+
+The following metrics are produced by `evaluate` stage. You can include or
+exclude them in `dvc exp show` with `--include-metrics` and `--exclude-metrics` options.
+
+- `categorical_accuracy`: Produces accuracy metrics for the classes.
+
+- `recall`: Recall metric (True Positives / All Relevant Elements)
+
+- `precision`: Precision metric (True Positives / All Positives)
+
+- `auc-roc`: Generates [Receiver Operating Characteristic](https://en.wikipedia.org/wiki/Receiver_operating_characteristic) curve
+
+- `auc-prc`: Generates Precision-Recall Curve
+
+- `fp`: Number of False Positives
+
+- `fn`: Number of False Negatives
+
+- `tp`: Number of True Positives
+
+- `tn`: Number of True Negatives
+
+## Files
+
+### Data Files
+
+The data files used in the project are found in `data/fashion-mnist`. All of
+these files are tracked by DVC and can be retrieved using `dvc pull` from the
+configured remote.
+
+- `data/fashion-mnist/raw.dvc`: Contains a reference to the [Dataset
+  Registry][dsr] to download the Fashion-MNIST
+  dataset to `data/fashion-mnist/raw/`.
+
+- `data/fashion-mnist/prepared/`: Created by `src/prepare.py` and contains training and testing files in NumPy format.
+
+- `data/fashion-mnist/preprocessed/`: Created by `src/preprocess.py` and contains training and
+  testing files in NumPy format ready to be supplied to `model.train`.
+
+### Source Files
+
+The source files are `src/` directory. All files receive runtime parameters from
+`params.yaml`, so none of them require any options. File dependencies are
+hardcoded in the current version, but this may change in a later iteration.
```

Comment:
this sounds like a note to self rather than info for users?

Reply:
Removed